### ISSUE TITLE
Accessibility score is 98/100 on Lighthouse

### ISSUE THREAD

**Issue by jpvalery** [narative/gatsby-theme-novela#140]
Been working on trying to optimize that and was wondering if it's something that you'd like to implement. 

Basically changing colors.ts to
```
import prism from "./prism";

export default {
  prism,

  primary: "#000",
  secondary: "#545454",
  grey: "#545454",
  background: "#fafafa",
  accent: "#373DD2",
  hover: "rgba(0, 0, 0, 0.07)",
  gradient: "linear-gradient(180deg, rgba(217, 219, 224, 0) 0%, #D9DBE0 100%)",
  articleText: "#08080B",
  track: "rgba(8, 8, 11, 0.3)",
  progress: "#000",
  card: "#fff",
  error: "#EE565B",
  success: "#46B17B",
  errorBackground: "rgba(238, 86, 91, 0.1)",
  horizontalRule: "rgba(8, 8, 11, 0.15)",
  inputBackground: "rgba(0, 0, 0, 0.05)",
  modes: {
    dark: {
      grey: "#545454",
      primary: "#fff",
      secondary: "#fff",
      accent: "#E9DAAC",
      background: "#16161D",
      hover: "rgba(255, 255, 255, 0.07)",
      gradient:
        "linear-gradient(180deg, #111216 0%, rgba(66, 81, 98, 0.36) 100%)",
      articleText: "#fff",
      track: "rgba(255, 255, 255, 0.3)",
      progress: "#fff",
      card: "#1D2128",
      error: "#EE565B",
      success: "#46B17B",
      errorBackground: "rgba(238, 86, 91, 0.1)",
      horizontalRule: "rgba(255, 255, 255, 0.15)",
      inputBackground: "rgba(255, 255, 255, 0.07)",
    },
  },
};
```

and updating the css of `MetaData` in `@narative/gatsby-theme-novela/src/sections/articles/Articles.List.tsx` to a higher opacity (couldn't test as shadowing didn't work on this, but I'm guessing that 0.55 should do the trick). 

I don't know if that's important to you or if you'd rather stick to the awesome design you made.

Here are the results on my fork: 
![image](https://user-images.githubusercontent.com/4112343/65079442-337cf280-d96d-11e9-9237-1dec538fe5c6.png)
I feel the changes are barely noticeable but would get the Accessibility score to 100. 

If you're ok with those changes, I could take of making a PR for that. Otherwise, this can be closed :)

**Comment by brotzky** [comment]
@tcostaco this is your territory.

We initially tried to keep the accessibility score high, but were okay with it not being 100/100 at the time of release.

**Comment by tcostaco** [comment]
Hey @jpvalery!

These are great suggestions. Although it's gonna be challenging getting 100/100 on both dark and light mode. Our current grey #73737D works very well in both colours, only failing in WCAG AAA (super rare case), whereas #545454 will drastically drop accessibility on dark mode to 2.47:1, failing in all levels. Unless I misunderstood the suggestions, I'd probably sacrifice these last 2%.

What do you think?

**Comment by jpvalery** [comment]
My bad, I thought I managed to run the Lightspeed audit on the dark mode as well. 🤦‍♂ 

As I said, it's not that important and can be easily overwritten by shadowing :) So if someone is striving to get a perfect score across the board, they can do it on their own. And anyways, 98/100 is a pretty respectable score 😛
